portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path

```diff
@@ -97,8 +173,41 @@
 	<td></td>
 </tr>
 
+<!-- LIST ITEM -->
+
+<tr>
+	<td>VIEW_LIST_ITEM</td>
+	<td>//tr[@data-testid='${key_itemValue}']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VIEW_LIST_ITEM_ORDER</td>
+	<td>xpath=(//td[@class='table-column-text-start'])[${index}]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASSING_MODAL_VIEW_LIST_ITEM_ORDER</td>
```

Comment:
Spelling: ASSIGN

Reply:
Fixed [here](https://github.com/liferay-ac/liferay-portal/pull/238/commits/2e593cb971771f0d8390534350cbb180adf00208). Thanks for catching this